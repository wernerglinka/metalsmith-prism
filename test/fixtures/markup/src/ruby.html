<code class="language-ruby">
  mystring = :steveT;
</code>
