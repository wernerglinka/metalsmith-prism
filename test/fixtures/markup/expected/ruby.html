<html><head></head><body><code class="language-ruby">
  mystring <span class="token operator">=</span> <span class="token symbol">:steveT</span><span class="token punctuation">;</span>
</code>
</body></html>
